src/mono/wasm/debugger/BrowserDebugProxy/EvaluateExpression.cs

```diff
@@ -424,6 +430,8 @@ private static object ConvertCSharpToJSType(object v, Type type)
                 return new { type = "object", subtype = "null", className = type?.ToString(), description = type?.ToString() };
             if (v is string s)
                 return new { type = "string", value = s, description = s };
+            if (v is char c)
```

Comment:
This method should be using the same methods to create these jobjects, as monosdbhelper. But it can be cleaned up in a follow up PR.

Reply:
Saved for later: https://github.com/dotnet/runtime/issues/66068